template_src/www/js/index.js

```diff
@@ -16,31 +16,14 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-var app = {
-    // Application Constructor
-    initialize: function() {
-        document.addEventListener('deviceready', this.onDeviceReady.bind(this), false);
-    },
 
-    // deviceready Event Handler
-    //
-    // Bind any cordova events here. Common events are:
-    // 'pause', 'resume', etc.
-    onDeviceReady: function() {
-        this.receivedEvent('deviceready');
-    },
+// Wait for the deviceready event before using any of Cordova's device APIs.
+// See https://cordova.apache.org/docs/en/latest/cordova/events/events.html#deviceready
+document.addEventListener('deviceready', onDeviceReady, false);
 
-    // Update DOM on a Received Event
-    receivedEvent: function(id) {
-        var parentElement = document.getElementById(id);
-        var listeningElement = parentElement.querySelector('.listening');
-        var receivedElement = parentElement.querySelector('.received');
+function onDeviceReady() {
+    // Cordova is now initialized. Have fun!
 
-        listeningElement.setAttribute('style', 'display:none;');
-        receivedElement.setAttribute('style', 'display:block;');
-
-        console.log('Received Event: ' + id);
-    }
-};
-
-app.initialize();
\ No newline at end of file
+    console.log('Running cordova-' + cordova.platformId + '@' + cordova.version);
```

Comment:
Just a nitpick... If IE is not supported we can use template literals.

```suggestion
    console.log(`Running cordova-${cordova.platformId}@${cordova.version}`);
```

Reply:
So this won't be a problem with the infamous Android 4.4?

Reply:
Ah yeah you right. Template literals is an ES6 thing and I believe Android 4.4 does not support ES6. I would say ignore this for now and merge as it.